docker/docker-compose.yml

```diff
@@ -16,9 +16,13 @@ services:
       - WEB_BASE_HREF=${WEB_BASE_HREF}
       - WEB_DEPLOY_URL=${WEB_DEPLOY_URL}
     networks:
+      - web
       - vcr
     ports:
       - "${WEB_HTTP_PORT}:8080"
 
 networks:
-  vcr:
\ No newline at end of file
+  web:
```

Comment:
Do we even need to declare a network, having only one component in this compose file?

Reply:
We should at least reference the external network

Reply:
Removed the default network, but still referencing `vcr` externally